cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +411,84 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("n3000", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
+				Skip("Cluster doesn't have intel fpga n3000 nic")
+			}
+		})
+
+		It("should have at least 1 intel n3000 nic", func() {
```

Comment:
should expose the fortville nics on the on the intel fpga pac N3000 as part of the n3000Node CR